src/leveldb-store.md

```diff
@@ -7,11 +7,13 @@ type: activemq5
 
 > **Warning**
 > 
-> The LevelDB store has been deprecated and is no longer supported or recommended for use. The recommended store is [KahaDB](kahadb)
+> The LevelDB store has been removed from production.
+> This page only serves as an archival page for previous releases.
+> Use [KahaDB](kahadb) instead.
 
 > **Version Compatibility**
 > 
-> Available in ActiveMQ 5.8.0 and newer
+> Available in ActiveMQ 5.8.0 until 5.17.0, not supported as of 5.14.2+
```

Comment:
Is this correct? Oh wait...I get it now. It will be removed after 5.17.0, but hasn't been supported since 5.14.2, even though it's still hung around. Consider clarifying here.

Reply:
I've tried to clarify this a bit - https://github.com/ehossack-aws/activemq-website/pull/2/commits/c5d4f885b00739e3633794c44c1f5f2ae46bdabc#diff-cbeca8b171e3a1c3158978f05d9900f10e7e40d2ccc09bd2b4902826a70e5127R16